### DISCUSSION TITLE
How find a memory consumer? This is no query.

### DISCUSSION THREAD

**Discussion by ShulcevVA** [ClickHouse/ClickHouse#44138] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/103301214/206936113-482b1808-8009-4a0e-be78-8b95b4e6adf1.png)

**Comment by den-crane** [comment, marked as answer]
check this https://kb.altinity.com/altinity-kb-setup-and-maintenance/altinity-kb-who-ate-my-memory/

**Reply by ShulcevVA** [reply]
Thank you very much!!!!!
